covid_graphs/covid_graphs/scripts/server.py

```diff
@@ -59,9 +59,34 @@ def covid19_predictions_redirect():
 def _run_flask_server(server: Flask, data_dir: Path):
     _create_prediction_apps(server=server, data_dir=data_dir)
     _create_simulation_apps(server=server, data_dir=data_dir)
+    _create_prediction_rest(server=server, data_dir=data_dir)
+    _create_graphs_app(server=server)
     server.run(host="0.0.0.0", port=8081)
 
 
+def _create_prediction_rest(data_dir: Path, server: Flask):
+    prediction_rest = country_dashboard.create_prediction_rest(data_dir=data_dir, server=server)
+
+    @server.route("/covid19/predictions/list/")
+    def covid19_available_predictions():
+        return prediction_rest.get_available_predictions()
+
+    @server.route("/covid19/predictions/data/<date>/<country>")
+    def covid19_get_specific_prediction(date, country):
+        return prediction_rest.get_specific_prediction(date, country)
+
+
+def _create_graphs_app(server: Flask):
+    # TODO (rejdi): It's better to use nginx for static files
+    @server.route("/covid19/predictions/graphs/")
+    def covid_19_graphs_index():
+        return send_file("../../../web/build/index.html", add_etags=True)
```

Comment:
Pri takýchto relatívnych cestách som trochu zhrozený. Flask má na statický obsah svoje mechanizmy https://exploreflask.com/en/latest/static.html, nedalo by sa to nejak použiť?

A inak možno by bolo dobré súbor `server.py` vybrať z balíka `covid_graphs` a presunúť ho do adresára `web`, kde bude všetko webové. @miskosz tuším aj také niečo spomínal, teda možno až na to oddelenie balíkov. Ale keďže teraz budeme mať asi časť webu v Reacte, tak by som dal webové veci spolu do jedného adresára.

To je ale dosť masívna úloha, takže to môžeme spraviť aj v ďalšom PR.

Reply:
Tak. Navyse to, co si poslal nie je moc dobre pouzitelne na to, co chceme. Best practice je postavit nginx pred flask. Statiku naservuje nginx, zlozitejsie veci presmeruje na flask. Riesil by tom to teda neskor v inom PR. Nateraz nachystam presun reactu do covid_graphs.